File: Pimo/Pimo/app/src/main/java/gram/com/pimo/SplashActivity.java
package gram.com.pimo;

import android.app.Activity;
import android.os.Bundle;
import android.os.Handler;
import android.os.Message;

/**
 * Created by dsm2017 on 2017-11-07.
 */

public class SplashActivity extends Activity
{
    @Override
    public void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.splash);
        initialize();
    }

    private void initialize()
    {
        Handler handler =    new Handler()
        {
            @Override
            public void handleMessage(Message msg)
            {
                finish();    // 액티비티 종료
            }
        };

        handler.sendEmptyMessageDelayed(0, 2000);    // ms, 3초후 종료시킴
    }
}

File: Pimo/Pimo/app/src/main/java/gram/com/pimo/Activity/SplashActivity.kt
package gram.com.pimo.Activity

import android.content.Intent
import android.os.Bundle
import gram.com.pimo.R
import gram.com.pimo.Util.BaseActivity

/**
 * Created by root1 on 2017. 11. 12..
 */

class SplashActivity: BaseActivity(){

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.splash)

        if(getPref().getString("token", "").isEmpty()){
            val intent = Intent(this, SignInActivity::class.java)
            startActivity(intent)
        }else{
            // val intent = Intent(this, MemoListActivity::class.java)
            val intent = Intent(this, MemoListActivity::class.java)
            startActivity(intent)
        }

        finish()
    }

}